website/versioned_docs/version-0.6.0/core/addresses.md

```diff
@@ -25,9 +25,9 @@ reason to keep using legacy transaction formats.
 ```scala
 // this generates a random private key
 val privkey = ECPrivateKey()
-// privkey: ECPrivateKey = Masked(ECPrivateKeyImpl)
+// privkey: ECPrivateKey = Masked(ECPrivateKey)
 val pubkey = privkey.publicKey
-// pubkey: org.bitcoins.crypto.ECPublicKey = ECPublicKey(02da06ea00abc1766e22267c3de60f8b878bee151640020e55b25e562364ee5d6e)
+// pubkey: org.bitcoins.crypto.ECPublicKey = ECPublicKey(039c31dc6d0670e4b12f82d847daacd0cc0b1e7ce66b3e80abdb060521ae9e30f4)
```

Comment:
the reason we have diffs here is because the JVM _literally_ runs this code. Thus we end up with randomly generated values now.